docs/agents/magpie.rst

```diff
@@ -122,9 +124,164 @@ To get lyrebird running, you must bring software up in the following order:
 
       run_lyrebird --port 8675 8676
 
-   and this will start lyrebird for the two corresponding slots.
+   and this will start lyrebird for the two corresponding slotkjs.
```

Comment:
Is this polish notation?

The --port arg still looks wrong, above.  Correct me...

Reply:
dziękuję, ya the `--port <port1> <port2>` works for me, and is I think is the correct usage for argparse `nargs='+'`